src/jobs/shellcheck.yml

```diff
@@ -0,0 +1,95 @@
+description: |
+  Automatically shellcheck the run statements within your jobs and commands.
+
+executor: cli/default
+
+parameters:
+  source-dir:
+    description: Path containing the orb source files.
+    type: string
+    default: "src"
+  exclude:
+    description: >
+      Enter a comma-separated list, with no spaces, of errors to ignore. Example: "SC1009,SC1073"
+    type: string
+    default: ""
+
+steps:
+  - checkout
+  - run:
+      name: Install shellcheck
+      command: |
+        if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+        $SUDO apt-get install shellcheck
+  - run:
+      name: Install YQ
+      command: |
+        if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+        wget https://github.com/mikefarah/yq/releases/download/3.2.1/yq_linux_386
+        $SUDO mv yq_linux_386 /usr/local/bin/yq
+        $SUDO chmod +x /usr/local/bin/yq
+  - run:
+      name: ShellCheck
+      command: |
+        set +e
+        # Set colors for terminal output
+        term_red=$'\e[1;31m'
+        term_green=$'\e[1;32m'
+        term_end=$'\e[0m'
+
+        ORB_SOURCE_FILES="<< parameters.source-dir >>/**/*.yml"
```

Comment:
This finds `.yml` files. We should cover the alt extension of `.yaml`.

Reply:
resolved